rosetta/transactions/parse_transaction.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package transactions
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go-sdk"
+
+	"github.com/optakt/flow-dps/rosetta/identifier"
+	"github.com/optakt/flow-dps/rosetta/object"
+)
+
+// ParseTransactions processes the flow transaction and translates it to a list of operations and a list of
+// signers.
+func (p *Parser) ParseTransaction(tx flow.Transaction) ([]object.Operation, []identifier.Account, error) {
```

Comment:
Used by other endpoints, here now to fully implement the interface.

Reply:
Transactions have a unique ID and should always be handled as pointers, not value types (`*flow.Transaction`).